src/components/Crud.test.tsx

```diff
@@ -0,0 +1,229 @@
+import { render, screen } from "@testing-library/react";
+import { Button, Crud, UserOption, filterUserList } from "./Crud";
+import { describe, expect, it, test, vi } from "vitest";
+import userEvent from "@testing-library/user-event";
+import { User, UserList } from "../pages/CrudPage";
+
+const user = userEvent.setup();
```

Comment:
deleted the const user = userEvent.setup(), because the library will clear up after test, when userEvent not be set up. And in my case, it is not necessary to set up.